src/services/cooperation.js

```diff
@@ -34,6 +34,10 @@ const cooperationService = {
 
     const cooperation = await Cooperation.findById(id)
 
+    if (price && status) {
```

Comment:
What happen if user change price to zero value and change the status concurrently? Was it tested?